js/cli_utils.ts

```diff
@@ -0,0 +1,93 @@
+import { TextEncoder } from "./text_encoding";
+
+const encoder = new TextEncoder();
+
+const tableChars = {
+  middleMiddle: "─",
+  rowMiddle: "┼",
+  topRight: "┐",
+  topLeft: "┌",
+  leftMiddle: "├",
+  topMiddle: "┬",
+  bottomRight: "┘",
+  bottomLeft: "└",
+  bottomMiddle: "┴",
+  rightMiddle: "┤",
+  left: "│ ",
+  right: " │",
+  middle: " │ "
+};
+
+const colorRegExp = /\u001b\[\d\d?m/g;
+
+function removeColors(str: string) {
+  return str.replace(colorRegExp, "");
+}
+
+const countBytes = (str: string) => {
```

Comment:
https://github.com/denoland/deno_std#top-level-functions-should-not-use-arrow-syntax

Also add return type.